lls/main.c

```diff
@@ -64,9 +116,24 @@ process_pkts(__attribute__((unused)) struct lls_config *lls_conf,
 			goto free_buf;
 
 		ether_type = eth_hdr->ether_type;
+		/*
+		 * Calculate offset to skip over VLAN header, if present.
+		 * TODO If VLAN stripping is enabled, we can skip this.
```

Comment:
I added a separate commit that cleans up some of the old `TODO` tags and adds one for VLANs. The stripping side should involve just a simple configuration change (following the assumption that current hardware supports our needs), but at least our current NIC does not support VLAN insertion, so that will need to be done in software.